sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -276,6 +276,8 @@ template <class T> struct ZeCache : private T {
 struct ReferenceCounter {
   ReferenceCounter(pi_uint32 InitVal) : RefCount{InitVal} {}
 
+  void reset(pi_uint32 InitVal) { RefCount = InitVal; }
```

Comment:
this is strange that it takes an arbitrary integer (I think you only call it with "1")
why is this needed?

Reply:
I agree, fixed.